lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -113,6 +114,14 @@ def post(self, request, *args, **kwargs):
         honor_mode = CourseMode.mode_for_course(course_key, CourseMode.HONOR)
         audit_mode = CourseMode.mode_for_course(course_key, CourseMode.AUDIT)
 
+        # Check to see if the User has an entitlement and enroll them if they have one for this course
+        if CourseEntitlement.check_for_existing_entitlement_and_enroll(user=user, course_run_key=course_key):
+            return JsonResponse(
+                {
+                    'redirect_destination': reverse('dashboard'),
```

Comment:
Does this mean we decided to keep the user being redirected to their dashboard even when they are enrolled in a run?

Reply:
additionally, I see that we changed it in views/management.py so I bet we want to update this as well

Reply:
Opps missed this one.

Reply:
I updated this.